src/redux/missions/missionSlice.js

```diff
@@ -0,0 +1,72 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+export const getMission = createAsyncThunk('mission/getMission', async () => {
+  try {
+    const resp = await axios.get(url);
+    const { data } = resp;
+    return data;
+  } catch (error) {
+    return error.message;
+  }
+});
+
+const initialState = {
+  mission: [],
+  isLoading: false,
+  error: undefined,
+};
+
+const missionSlice = createSlice({
+  name: 'mission',
+  initialState,
+  reducers: {
+    // missionAdded(state, action) {
+    //   console.log(action);
+    //   const newArr = state.mission.concat(action.payload);
+    //   console.log(newArr);
+    //   return { ...state, mission: newArr };
+    // },
```

Comment:
It would be nice to remove unused code for it can cause errors and also it makes your code base look untidy please lets acknowledge microverse clean code policy.